qp_klp/Step.py

```diff
@@ -607,6 +613,58 @@ def load_preps_into_qiita(self, qclient):
 
         return df
 
+    def _load_preps_into_qiita2(self, qclient, prep_id, qiita_id, out_dir,
+                               project):
+        surl = f'{qclient._server_url}/study/description/{qiita_id}'
+        prep_url = (f'{qclient._server_url}/study/description/'
+                    f'{qiita_id}?prep_id={prep_id}')
+
+        if self.pipeline.pipeline_type == Step.AMPLICON_TYPE:
+            files = self._get_files_amplicon(out_dir, project)
+        else:
+            atype = 'per_sample_FASTQ'
+            files = self._get_files_meta(out_dir, project, prep_id)
+
+        for f_type in files:
+            if not files[f_type]:
```

Comment:
it looks like this is a `dict`? if so then suggest:

```suggestion
        for f_type, size in files.items():
            if size == 0:
```

Reply:
...althoguh I don't understand the variable name `f_type`?

Reply:
thanks! size == 0 is easier to read. I changed f_type to 'key' because that's really what they are - we're just checking the values associated w/each key to make sure they're not empty.